docs/source/quark_script.rst

```diff
@@ -1353,3 +1359,64 @@ Quark Script Result
             and it occurs in method, Lowasp/sat/agoat/AccessControlIssue1Activity; hashPIN (Ljava/lang/String;)Ljava/lang/String;
     CWE-328 is detected in ./MSTG-Android-Java.apk,
         and it occurs in method, Lcom/tozny/crypto/android/AesCbcWithIntegrity; generateKeyFromPassword (Ljava/lang/String; [B)Lcom/tozny/crypto/android/AesCbcWithIntegrity$SecretKeys;
+
+Detect CWE-295 in Android Application (InsecureShop.apk)
+----------------------------------------------------------
+
+This scenario seeks to find **Improper Certificate Validation**. See
+`CWE-295 <https://cwe.mitre.org/data/definitions/295.html>`__ for more
+details.
+
+Let’s use this `APK <https://github.com/hax0rgb/InsecureShop>`__ and the
+above APIs to show how the Quark script finds this vulnerability.
+
+We use the API ``findMethodInAPK`` to locate all
+``SslErrorHandler.proceed`` methods. Then we need to identify whether if
+the method ``WebViewClient.onReceivedSslError`` is overrode by its
+subclass.
+
+First, we check and make sure that the ``MethodInstance.name`` is
+``onReceivedSslError``, and the ``MethodInstance.descriptor`` is
+``(Landroid/webkit/WebView; Landroid/webkit/SslErrorHandler; Landroid/net/http/SslError;)V``.
+
+Then we use the method A
+``MethodInstance.findSuperclassHierarchy``\ to get the supclass list of
+the method’s caller class.
+
+Finally, we check the ``Landroid/webkit/WebViewClient;`` is on the
+supclass list. If **YES** , that may cause CWE-295 vulnerability.
+
+Quark Script CWE-295.py
+========================
+
+.. code:: python
+
+   from quark.script import findMethodInAPK
+
+   SAMPLE_PATH = "insecureShop.apk"
+   TARGET_METHOD = [
+       "Landroid/webkit/SslErrorHandler;",  # class name
+       "proceed",                          # method name
+       "()V"                               # descriptor
+   ]
+    OVERRIDE_METHOD = [
+        "Landroid/webkit/WebViewClient;",  # class name
+        "onReceivedSslError",              # method name
+        # descriptor
+        "(Landroid/webkit/WebView; Landroid/webkit/SslErrorHandler; Landroid/net/http/SslError;)V"
```

Comment:
I suggest putting this comment at the end of line 1406. This way, the comment style used for this line is consistent with the above lines.